src/routes/api/contest.route.js

```diff
@@ -1,7 +1,8 @@
 const express = require('express');
 const router = express.Router()
```

Comment:
move this below require statements. its better to group them at the top like importing packages in java

Reply:
interesting that the linter didn't catch this

Reply:
eslint isn't configured properly

Reply:
You need to add the eslint extension that checks imports

it's called eslint-plugin-import

https://www.npmjs.com/package/eslint-plugin-import